relnotes/fiber-pool.feature.md

```diff
@@ -0,0 +1,30 @@
+* `ocean.task.Scheduler`
+
+  It is now possible to define dedicated worker fiber pools to process tasks of
+  specific type:
+
+  ```D
+  SchedulerConfiguration config;
+
+  with (config)
+  {
+    specialized_pools = [
+      PoolDescription(MyTask.classinfo, 10240),
+      PoolDescription(MyOtherTask.classinfo, 2048),
+    ];
+  }
+
+  initScheduler(config);
+
+  // will be processed immediately in own worker fiber pool, skipping the queue:
+  theScheduler.schedule(new MyTask);
+  ```
+
+  Such functionality is intended to be used when there are some task types that
+  need real-time handling and thus can't afford waiting in the queue. It can
+  also be used for case when some tasks are very long-living would hold worker
```

Comment:
Grammar / clarity: "It can also be used for the case where some very long-lived tasks would occupy worker fibers from the shared pool forever, if processed normally."